xla/stream_executor/cuda/cuda_executor.cc

```diff
@@ -766,10 +766,6 @@ absl::StatusOr<std::unique_ptr<GpuEvent>> GpuExecutor::CreateGpuEvent(
   return std::move(gpu_event);
 }
 
-absl::StatusOr<std::unique_ptr<Event>> GpuExecutor::CreateEvent() {
```

Comment:
I also pushed this after the approval to fix the last remaining linking error. My understanding of what was going on is that `GPUExecutor` is not an abstract class as it's sometimes instantiated in the codebase directly. This means that it needs to implement `CreateEvent` which is a pure virtual function. Since both the CUDA and the ROCm executors used the same implementation, I moved it to `GPUExecutor` this resolved the linking error. Without this change, `cuda_executor.obj` was ending up with a symbol named `CreateEventA` and the linker was failing for places in the code that needed to link to `CreateEvent`. I'm not quite sure how that `A` ends up in there and what the name mangling scheme is since I'm not familiar with C++, but that's my best attempt at an explanation of what was going on.